pkgs/top-level/python-packages.nix

```
@@ -4302,7 +4306,10 @@ in {
 
   paperspace = callPackage ../development/python-modules/paperspace { };
 
+  openpaperwork-core = callPackage ../applications/office/paperwork/opencore.nix { };
+  openpaperwork-gtk = callPackage ../applications/office/paperwork/opengtk.nix { };
```

```suggestion
  openpaperwork-core = callPackage ../applications/office/paperwork/core.nix { };
  openpaperwork-gtk = callPackage ../applications/office/paperwork/gtk.nix { };
```

Although what about exposing all these parts behind a single paperwork attribute set instead, eg. `paperwork.core`, `paperwork.gkt`, ...?

---

I wanted one derivation per package because it plays nice with buildPythonPackage.
I wanted one attr per derivation because this way overriding is easier.

---

Okay, but still, I find `opencore` a bit confusing, so why not just call those files `core.nix` and `gtk.nix`? IMO, the `open` doesn't really add anything in value.

---

there is openpaperwork-gtk and paperwork-gtk, and they are different...

---

Okay, where do I find the latter and what's the difference between them?

---

See the top-level directory of https://gitlab.gnome.org/World/OpenPaperwork/paperwork for the source
paperwork-gtk is default.nix and openpaperwork-gtk is in opengtk.nix
The latter is supposed to only contain reusable components.

---

Ah, so given that the `openpaperwork` components are to be used outside of Paperwork, what about moving it into a different directory or even into `pkgs/development/python-modules/openpaperwork-*`?

---

I renamed files to the full package name, to minimize ambiguity.